src/lang/en.json

```diff
@@ -677,7 +677,9 @@
                     "Cost": "Activation Cost",
                     "Consume": "Resource Consumption",
                     "Uses": "Limited Uses",
-                    "Recharge": "Recharge"
+                    "Recharge": "Recharge",
+                    "AdditionalFormula": "Roll Formula Extension",
+                    "AdditionalFormulaDescription": "Input any extra modifiers you wish to add to this item's d20 Skill Roll. This input will be treated as a dice formula, but any die you add won't be available for advantage/disadvantage currently. Paramters you can use: @mod, @attribute & @skill.rank refer to the values of the skills selected here, but the whole list of skills and attributes can be accessed - ask on the discord for help!"
                 },
```

Comment:
I think it's preferrable for this to be an on-hover tooltip rather than a description that's always visible. It buys us some real estate in the sheeet

Reply:
I don't disagree, I just don't know if we use tooltips for providing info like that elsewhere in any of the sheets yet, do we?
This is where something like a set of the tutorials/walkthroughs ( or a wiki

Reply:
I'm all for moving any hint infos to tooltips wherever we can. It's just cleaner UI with less overwhelming amounts of text